BitwardenShared/Core/Vault/Models/Enum/PolicyOptionType.swift

```diff
@@ -10,7 +10,7 @@ enum PolicyOptionType: String {
     case capitalize
 
     /// A policy option for the default type of the password generator.
-    case defaultType
+    case overridePasswordType
```

Comment:
⛏️ The documentation comment should probably be updated as well

Reply:
updated!